modules/customize-certificates-understanding-service-serving.adoc

```diff
@@ -15,5 +15,6 @@ algorithm to generate service certificates.
 The generated certificate and key are in PEM format, stored in `tls.crt`
 and `tls.key` respectively, within a created secret. The
 certificate and key are automatically replaced when they get close to
-expiration. The service CA certificate, which signs the service
-certificates, is only valid for one year after {product-title} is installed.
+expiration.
+
+The service CA certificate, which issues the service certificates, is valid for 26 months and is automatically rotated when there is less than six months validity left. After rotation, the previous service CA configuration is still trusted until its expiration. This allows a grace period for all affected services to refresh their key material before the expiration. If you do not upgrade your cluster during this grace period, which restarts services and refreshes their key material, you might need to manually restart services to avoid failures after the previous service CA expires.
```

Comment:
We need to link to https://docs.openshift.com/container-platform/4.3/authentication/certificates/service-serving-certificate.html#manually-rotate-service-ca_service-serving-certificate directly. 

Telling people the process they need to follow as part of this call out is critical.

Reply:
@sferich888 Unfortunately I can't link right now due to our current guidelines.

And @marun can confirm, but they shouldn't need to follow that whole procedure (since rotation is handled during the upgrade), but would just need to do step 3 to restart the pods. 

But I can add that step's info right here too.

Reply:
Ok this works for me; however, we should include the other module here vs duplicating content?